setup.py

```diff
@@ -36,6 +36,7 @@
 
 scripts = ['shell/activate',
            'shell/deactivate',
+           'shell/conda.fish' 
```

Comment:
Nah, `setup.py` can't really handle it correctly.  `scripts` would put `conda.fish` in the `./bin` directory, which is not quite where we want it.  Conda's own conda-build recipe puts `shell/conda.fish` at `$PREFIX/etc/fish/conf.d/conda.fish`, which I think is the "correct" place for it to go.

https://github.com/conda/conda/blob/master/conda.recipe/build.sh#L10-L11